xiplot/plugin.py

```diff
@@ -88,7 +91,8 @@ def placeholder_figure(text: str) -> Dict[str, Any]:
 def get_plugins_cached(
```

Comment:
Caching the lists of plugins is a premature optimisation (unless we need it for keeping track of existing callbacks). So feel free to redo this function if necessary.